package.json

```diff
@@ -17,6 +17,7 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-router-dom": "^6.10.0",
+    "react-scroll": "^1.8.9",
```

Comment:
would you mind explaining me this lib?

Reply:
wanted to try a scroll effect but kicked it out - thanks, will take it out of the package json!